packages/video_player/lib/video_player.dart

```diff
@@ -173,14 +174,15 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
         value = value.copyWith(isPlaying: false);
         timer?.cancel();
       } else if (map["event"] == "bufferingUpdate") {
-        final List<List<int>> bufferedValues = map["values"];
+        final List<dynamic> bufferedValues = map["values"];
```

Comment:
You could consider using 

List<List<dynamic>> bufferedValues = map["values"].cast();

if you wanted to preserve more of the types (and if you are able to use .cast which is available starting in dev.21).  This would allow toDurationRange to be typed to take List<dynamic>.

Reply:
When I made the change suggested by you
List<List> bufferedValues = map["values"].cast<List>();
I get the following analyzer errors  
info - Specify type annotations at lib/video_player.dart:176:20 - (always_specify_types)
  info - Specify type annotations at lib/video_player.dart:176:62 - (always_specify_types)
not sure what to make of it.

Reply:
I'm not too familiar with always_specify_types, but I'd bet it requires you to write List<dynamic> explicitly instead of just List (and similarly List<List<dynamic>> instead of List<List>).